pkg/validation/validation.go

```diff
@@ -0,0 +1,83 @@
+package validation
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ghodss/yaml"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+// https://github.com/xeipuuv/gojsonschema#formats
+type ValidFormat struct{}
+
+// IsFormat always returns true and meets the
+// gojsonschema.FormatChecker interface
+func (f ValidFormat) IsFormat(input interface{}) bool {
+	return true
```

Comment:
Sholudn't this do something? This looks weird

Reply:
Reference: https://github.com/garethr/kubeval/blob/master/kubeval/kubeval.go#L42